### PR TITLE
14172- Update "How does it work?"

### PR DESCRIPTION
**Pull request by ketaki-deodhar**
*Issue #:*
https://github.com/bcgov/entity/issues/#14172

*Description of changes:*
Based on Yui's comment, added a tooltip for "BC based Company" expansion panel.  I had to separate the expansion panels as only one has tooltip.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the sbc-auth license (Apache 2.0).

### CODE REVIEW

File: auth-web/package-lock.json [hunk 1]
```diff
@@ -47,7 +47,7 @@
         "@types/vuelidate": "^0.7.13",
         "@typescript-eslint/eslint-plugin": "^5.23.0",
         "@typescript-eslint/parser": "^5.23.0",
-        "@volar-plugins/vetur": "*",
+        "@volar-plugins/vetur": "latest",
```
Comment (severinbeauvais) [inline auth-web/package-lock.json:50]: Did you change this?

File: auth-web/src/views/auth/home/IncorpOrRegisterView.vue [hunk 2]
```diff
@@ -38,31 +38,65 @@
...
+                  </v-expansion-panel-header>
+                  <v-expansion-panel-content class="list-item-text py-3">
+                    <span id="expPnlContent3">Office addresses, director names and addresses, rules of the association
+                      and memorandum.</span>
                   </v-expansion-panel-content>
                 </v-expansion-panel>
               </v-expansion-panels>
             </div>
+
+            <!-- For BEN only as feature flag 'EnableBcCccUlc' enabled. Remove section when FF removed -->
```
Comment (severinbeauvais) [inline auth-web/src/views/auth/home/IncorpOrRegisterView.vue:99]: Did you create a tech debt ticket to remove the obsolete code and feature flag when this feature is released to Prod?

If so then I think you don't need the extra comment here.

File: auth-web/package-lock.json (hunk 1 above)
Comment (ketaki-deodhar) [reply auth-web/package-lock.json:50]: I did not

File: auth-web/src/views/auth/home/IncorpOrRegisterView.vue [hunk 3]
```diff
@@ -38,31 +38,65 @@
...
+                    <template v-slot:actions>
+                      <v-icon color="primary">
+                        $expand
+                      </v-icon>
+                    </template>
+                    Cooperative Association
+                  </v-expansion-panel-header>
+                  <v-expansion-panel-content class="list-item-text py-3">
+                    <span id="expPnlContent3">Office addresses, director names and addresses, rules of the association
+                      and memorandum.</span>
```
Comment (severinbeauvais) [inline auth-web/src/views/auth/home/IncorpOrRegisterView.vue:93]: Can you please provide before and after screenshots? They aren't always neeed but in this case they'd help me a lot :)

File: auth-web/src/views/auth/home/IncorpOrRegisterView.vue (hunk 2 above)
Comment (ketaki-deodhar) [reply auth-web/src/views/auth/home/IncorpOrRegisterView.vue:99]: I don't think I have yet but will do right away

File: auth-web/package-lock.json (hunk 1 above)
Comment (severinbeauvais) [reply auth-web/package-lock.json:50]: Weird. I just added the package to Filings UI (to see what it does) and it looks like this for me:
```
"@volar-plugins/vetur": "^0.1.0",
```

Comment (severinbeauvais) [reply auth-web/package-lock.json:50]: @seeker25 What's this package about, and why did the spec change?

File: auth-web/src/views/auth/home/IncorpOrRegisterView.vue (hunk 2 above)
Comment (ketaki-deodhar) [reply auth-web/src/views/auth/home/IncorpOrRegisterView.vue:99]: Before:
image


After:
image

File: auth-web/package-lock.json (hunk 1 above)
Comment (lambert-alex) [reply auth-web/package-lock.json:50]: I think its a problem with you npm config files.

File: auth-web/src/views/auth/home/IncorpOrRegisterView.vue (hunk 3 above)
Comment (ketaki-deodhar) [reply auth-web/src/views/auth/home/IncorpOrRegisterView.vue:93]: Before:
image


After: somehow cursor not showing in capture
image

File: auth-web/package-lock.json (hunk 1 above)
Comment (ketaki-deodhar) [reply auth-web/package-lock.json:50]: just ran 'npm i'. You think reverting back to the way it was good idea?

File: auth-web/src/views/auth/home/IncorpOrRegisterView.vue (hunk 3 above)
Comment (severinbeauvais) [reply auth-web/src/views/auth/home/IncorpOrRegisterView.vue:93]: Nice. Looks just like the design :)

File: auth-web/package-lock.json (hunk 1 above)
Comment (severinbeauvais) [reply auth-web/package-lock.json:50]: Let's see what Travis knows about this. In the meantime, I'd be tempted to revert this line.

Comment (ketaki-deodhar) [reply auth-web/package-lock.json:50]: I will wait for Travis's reply :)
I am on vacation starting tomorrow but will check this sometime tomorrow and will take it from there.

Comment (seeker25) [reply auth-web/package-lock.json:50]: I changed it, was recommendations straight out of VSCODE.

Comment (severinbeauvais) [reply auth-web/package-lock.json:50]: Oh, OK. So Ketaki is missing a rebase.

**Comment by seeker25** [comment]
I've created a PR to revert old changes, just so we can push the build out. https://github.com/bcgov/sbc-auth/pull/2190

**Comment by severinbeauvais** [comment]
> @seeker25 @severinbeauvais, I will create a new branch as HEAD is way ahead and I do not want to risk loosing any incoming changes. I hope it is okay to close this PR and create a new one based on my new branch.

Sounds good.

**Pull request closed without merging**